versions-maven-plugin/src/it/it-set-issue-848/pom.xml

```diff
@@ -0,0 +1,15 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>default-group</groupId>
+    <artifactId>default-artifact</artifactId>
+    <version>1.0-SNAPSHOT</version>
+    <packaging>pom</packaging>
+
+    <modules>
+        <module>moduleA/moduleA.xml</module>
```

Comment:
Please add something  like:

```
<module>moduleA/moduleA1.xml</module>
<module>moduleA/moduleA2.xml</module>
```

Reply:
Fix for it should be in `org.codehaus.mojo.versions.api.PomHelper#getReactorModels`

We have:
```
String modulePath = (!path.isEmpty() && !path.endsWith("/") ? path + "/" : path)
                            + pomFile.getParentFile().getName();
```

So we drop info info about pom name in the same directory.

Reply:
Good point! Thanks.